app/javascript/javascript/allocation_decision.js

```diff
@@ -0,0 +1,11 @@
+function selectMotCode() {
```

Comment:
If this isn't hooked up to the html/erb file, do you want to remove it? Or work out how to get it to work via this extracted out structure?